debian/patches/0027-add-fixes-to-improve-the-performance-of-vaapi-encode.patch

```diff
@@ -209,25 +218,29 @@ Index: jellyfin-ffmpeg/libavcodec/vaapi_encode.h
  
      // Slice structure.
      int slice_block_rows;
-@@ -345,6 +348,10 @@ typedef struct VAAPIEncodeContext {
+@@ -345,6 +348,13 @@ typedef struct VAAPIEncodeContext {
      int             roi_warned;
  
      AVFrame         *frame;
 +
++    // Whether the driver support vaSyncBuffer.
++    int             has_sync_buffer_func;
```

Comment:
```suggestion
+    // Whether the driver supports vaSyncBuffer.
+    int has_sync_buffer_func;
```

Reply:
copied from https://github.com/FFmpeg/FFmpeg/commit/e0ff86993052b49a64d434bac345e92fc149f446 i think we should leave it as is.